native_client/deepspeech.h

```diff
@@ -163,9 +163,12 @@ int DS_EnableExternalScorer(ModelState* aCtx,
 /**
  * @brief Add a hot-word and its boost.
  *
+ * Excessive positive boost might lead to splitting up of letters of the word following the hot-word.
```

Comment:
This part also should be moved at `@param`

Reply:
Okay, I see, makes sense.